src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -59,8 +59,8 @@ public function __construct($queryBuilder, $manager = null, $class = null)
         if ($queryBuilder instanceof \Closure) {
             @trigger_error('Passing a QueryBuilder closure to '.__CLASS__.'::__construct() is deprecated since version 2.7 and will be removed in 3.0.', E_USER_DEPRECATED);
 
-            if (!$manager instanceof EntityManager) {
-                throw new UnexpectedTypeException($manager, 'Doctrine\ORM\EntityManager');
+            if (!$manager instanceof EntityManagerInterface) {
```

Comment:
EntityManagerInterface is only available since ORM 2.4. So you must check against both to keep functionality with ORM 2.3. Btw this code path is deprecated anyway. So it might not be worth adding features for that.